### PR TITLE
WIP - First iteration on improving existing docs

### PR DESCRIPTION
**Pull request by iznatalia**
In order to decrease amount of incoming tickets regarding iZettle public API we need to improve existing docs that are not exhaustive and clearly not very pleasant to read.
 
- Structure API docs
- Make doc pages more readable
- Add more examples
- Spellcheck/corrections in content

### CODE REVIEW

File: finance.adoc [hunk 1]
```diff
@@ -31,6 +31,7 @@ https://finance.izettle.com
 *Example responses:*
 
 `GET /organizations/self/accounts/LIQUID/balance`
```
Comment (Astro-Choco) [inline finance.adoc:33]: I think this line belongs to example requests.

File: finance.adoc [hunk 2]
```diff
@@ -31,6 +31,7 @@ https://finance.izettle.com
 *Example responses:*
```
Comment (Astro-Choco) [inline finance.adoc:31]: as we have already used a header styling, a colon ":" is not needed.

File: faq.adoc [hunk 3]
```diff
@@ -1,57 +1,73 @@
...
-****
-'''
+====
+If you want to apply to receive access to our APIs, please visit https://developer.izettle.com/register[https://developer.izettle.com/register]
+====
 
-=== General questions
-**What TLS version is supported?**
+== General questions
+.*_What TLS version is supported?_*
```
Comment (Astro-Choco) [inline faq.adoc:9]: Usually we don't use italic style for regular text. So what about only using a header styling (bold) here?
The same comment for other FAQ questions.

File: faq.adoc [hunk 4]
```diff
@@ -60,13 +76,16 @@ Example response
...
 --
 See https://github.com/iZettle/api-documentation/blob/master/authorization.adoc[OAuth] for more info.
+****
+====
 
-'''
-
-=== https://github.com/iZettle/api-documentation/blob/master/product-library.adoc[Product Library]
+== Product Library
+.*_I'm using your API to create Products on iZettle, i have a problem when assigning images to products, i used the image API to create an image and it's working, but how to assign this image to the product i'm creating?_*
```
Comment (Astro-Choco) [inline faq.adoc:83]: There're many writing typos and wrong styles in the existing docs like this here. I can fix them in another PR.

File: finance.adoc (hunk 2 above)
Comment (iznatalia) [reply finance.adoc:31]: I changed this to just **Examples** after our meeting today.
Can you please review? 🙏

File: finance.adoc [hunk 5]
```diff
@@ -120,25 +149,54 @@ offset (optional):: Offset the result set by X number of results.
...
+
+[%collapsible]
+====
+**Parameters:**
+
+[grid="none", frame="none" cols="20%,10%,10%,10%,50%"]
+|===
+|Name|Type|In|Required/Optional|Description
+
+|`organizationUUID`|_string_|_path_|*_required_*|Organization identifier as UUID or `self to denote that the organization should be derived from the authenticated user.
```
Comment (Astro-Choco) [inline finance.adoc:189]: As this info doesn't require special styling, I'd think regular styling without bold and italic is good enough.
The same comment goes to similar occurrences in the reference tables.

File: image.adoc [hunk 6]
```diff
@@ -1,43 +1,62 @@
...
+== Description
+====
+Image API allows upload of product images to be stored and used within iZettle applications.
+The API returns an `imageUrl` that can be added to a desired product using the Product Library API.
+Images can be uploaded using a byte array or a static URL, the image will be eventually available for look up from
+within iZettle apps or via `imageUrls` returned by the API.
 
-### URL
-https://image.izettle.com/v2/images
+NOTE: JPG does not exist as `imageFormat`, you must use "JPEG" for .jpg images.
```
Comment (Astro-Choco) [inline image.adoc:11]: **Note:**
Question: why is this note displayed in table cells? Sorry I didn't figure out what tagging creates table cells here.

**Review by Astro-Choco — COMMENTED** [review]
Very good changes for readability. Thanks!
I made a few comments on the high level style.

File: finance.adoc (hunk 2 above)
Comment (Astro-Choco) [reply finance.adoc]: Yes, it looks great with what you've changed. :D
I like your structure:
**Example** 
then **Request** 
and then **Response**

File: image.adoc (hunk 6 above)
Comment (iznatalia) [reply image.adoc:11]: I think Github styles it differently.

File: giftcard.adoc [hunk 7]
```diff
@@ -1,46 +1,74 @@
...
+|`uuid`|string|path|required|Unique id as `UUID`.
+|===
+
 
-uuid:: The UUID of the giftcard
+*Examples*
 
-#### Errors
-404:: Giftcard not found
+1. Fetch gift card information by uuid.
```
Comment (Astro-Choco) [inline giftcard.adoc:29]: Is this a step?
If not, I'd use a unnumbered bullet point.

File: giftcard.adoc [hunk 8]
```diff
@@ -1,46 +1,74 @@
...
+==== Fetch gift card
 
 `GET organizations/self/giftcards/external/{uuid}`
 
-#### Permissions required
-`READ:PURCHASE`
+|===
+|Name|Type|In|Required/Optional|Description
+
+|`uuid`|string|path|required|Unique id as `UUID`.
```
Comment (Astro-Choco) [inline giftcard.adoc:23]: Is UUID or uuid?
In parameter tables, no special styling is needed. So I'd remove ``.
Do users know where they can find the uuid? If not, I'd add where to find it.
Usually when we enter credit card number at the payment, we're asked to enter security code. Not everyone knows what security code is and where to find it. So there's usually an instruction about where we can see the code on our cards.

File: finance.adoc [hunk 9]
```diff
@@ -120,25 +159,54 @@ offset (optional):: Offset the result set by X number of results.
...
+|`at`|string|query|optional|Used to get payouts until a specific historic date. Formatted as an `ISO 8601` string.
+|===
+
+*Examples*
+
+1. Fetching payout info.
++
+--
+
+_Example request:_
```
Comment (Astro-Choco) [inline finance.adoc:201]: Write Request to be consistent with other docs. The same comment applies to similar occurrences.

File: giftcard.adoc [hunk 10]
```diff
@@ -1,46 +1,74 @@
...
-### Service Description
-The Giftcard API provides extra information about purchases made through iZettle with a giftcard.
+=== Description
+====
+Giftcard API provides extra information about purchases made through iZettle with a gift card.
+====
 
-### URL
+=== URL
 https://giftcard.izettle.com
```
Comment (Astro-Choco) [inline giftcard.adoc:9]: If this is a base URL, I'd write Base URL. The same comment applies to similar occurrences.

File: giftcard.adoc [hunk 11]
```diff
@@ -1,46 +1,74 @@
-## Giftcard API
+== Giftcard API
```
Comment (Astro-Choco) [inline giftcard.adoc:1]: As you used later, the header should be Gift Card API.

**Review by Astro-Choco — COMMENTED** [review]
Very good improvements. :) Just minor stuff.

**Review by Astro-Choco — COMMENTED** [review]
Just a question about OpenAPI specs.

File: CHANGELOG.adoc [hunk 12]
```diff
@@ -1,44 +1,46 @@
...
 
-### 2020-10-28
-
-#### Apply a Beta label on any mention of API documentation
+.**Oct 28, 2020** `Editorial changes`
+[%collapsible]
+====
+**Apply a Beta label on any mention of API documentation**
 
 The current state of the API documentation leave plenty of room for improvement. We feel that in order to be transparent and set expectations right, we've applied a Beta label on any mention of the API documentation. In the meantime, we will be hard at work at improving the iZettle Developer Platform. Some examples of the ongoing work includes revisiting all OpenAPI (Swagger) specifications and making sure that no details are left out, and make more richer documentation available to the wider developer audience.
```
[the remark this answers is not in the document]
Comment (iznatalia) [reply CHANGELOG.adoc:8]: No it's quite new. The text is basically saying that we are to improve our doc using OpenAPI specification and probably Swagger UI. But i agree "revisiting" sounds a bit confusing. 🤔

[the remark this answers is not in the document]
Comment (iznatalia) [reply CHANGELOG.adoc:8]: Feel free to change later on 👍

**Review by jrclaramunt-izettle — COMMENTED** [review]
Finance part looks good 🏅

**Review by Astro-Choco — DISMISSED** [review]
Very good improvements, especially how to use `linkUrls`. 😀

**Comment by fbnkadev1** [comment]
Hello,

we are using the https://finance.izettle.com API, specifically the operation:

`GET /organizations/self/accounts/PRELIMINARY/transactions`

to obtain the transactions made with the iZettle dataphones that we have in our restaurant.

We have created several subaccounts in the iZettle control panel, with the aim that each employee has their own dataphone and that the transactions carried out are associated with a subaccount at all times, one for each employee.

However, when invoking the RESTAPI operation indicated above, we do not know how to obtain the transactions filtered by subaccount.

Is there a way to do it?

Could the operation to obtain the transactions (https://github.com/iZettle/api-documentation/blob/reformat-docs/finance.adoc#fetch-account-transactions) return the associated subaccount for each transaction?

Otherwise, are there other API operations that allow us to obtain the following information?
- Obtain the list of subaccounts registered.
- Consult the PRELIMINARY transactions for a specific subaccount.

We await your news.
Best regards.

**Pull request closed without merging**